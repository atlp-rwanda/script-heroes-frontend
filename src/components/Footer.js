import React from 'react'
import {Container, Row, Col} from 'reactstrap'
import facebook from '../assets/fb.png'
import twitter from '../assets/twitter.png'
import linkedin from '../assets/linkedin.png'
import instagram from '../assets/insta.gif'

export default function Footer() {
    return (
        <Container fluid className='footer'>
            <Row className="just-foot">
                <Row className='social-links'>
                    <Col>
                        <a href='https://www.facebook.com/' target='_blank'><img src={facebook}/></a>
                        <a href='http://instagram.com/' target='_blank'><img src={instagram}/></a>
                        <a href='http://linkedin.com/' target='_blank'><img src={linkedin}/></a>
                        <a href='https://twitter.com/' target='_blank'><img src={twitter}/></a> 
                    </Col>
                </Row>
                <Row>
                    <Col>
                        <p>&copy; 2020 Barefoot Nomad</p> 
                    </Col>
                </Row>
            </Row>
        </Container>
    )
}
