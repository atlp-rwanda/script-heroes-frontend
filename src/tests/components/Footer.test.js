import {shallow} from 'enzyme'
import toJSON from 'enzyme-to-json'
import React from 'react'
import Footer from '../../components/Footer'

describe("Footer", ()=>{
  it("Should match snapshot", ()=>{
    const wrapper= shallow(<Footer/>)
    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})