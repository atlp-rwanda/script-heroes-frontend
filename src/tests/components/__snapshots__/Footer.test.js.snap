// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Should match snapshot 1`] = `
<div
  className="footer-containter d-flex"
>
  <div
    className="social"
  >
    <a
      href="https://www.facebook.com/"
      target="_blank"
    >
      <img
        src=""
      />
    </a>
    <a
      href="http://instagram.com/"
      target="_blank"
    >
      <img
        src=""
      />
    </a>
    <a
      href="http://linkedin.com/"
      target="_blank"
    >
      <img
        src=""
      />
    </a>
    <a
      href="https://twitter.com/"
      target="_blank"
    >
      <img
        src=""
      />
    </a>
  </div>
  <p>
    Â© 2020 Barefoot Nomad
  </p>
</div>
`;
